File: Computer-Seekho-Java/ComputerSeekho/src/main/java/com/Project/Entities/Enquiry.java
package com.Project.Entities;

import java.time.LocalDate;
import jakarta.persistence.*;
import lombok.Data;

@Entity
@Table(name = "enquiry")
@Data
public class Enquiry {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "enquiry_id")
    private int enquiryId;

    @Column(name = "enquirer_name")
    private String enquirerName;

    @Column(name = "enquirer_address")
    private String enquirerAddress;

    @Column(name = "enquirer_mobile")
    private String enquirerMobile;

    @Column(name = "enquirer_email_id")
    private String enquirerEmailId;

    @Column(name = "enquiry_date")
    private LocalDate enquiryDate;

    @Column(name = "enquirer_query")
    private String enquirerQuery;

    @Column(name = "course_name")
    private String courseName;

    @ManyToOne
    @JoinColumn(name = "staff_id")
    private Staff staff;

    @Column(name = "student_name")
    private String studentName;

    @Column(name = "enquiry_counter")
    private int enquiryCounter;

    @Column(name = "closure_reason")
    private String closureReason;

    @Column(name = "follow_up_date")
    private LocalDate followUpDate;

    @Column(name = "enquiry_is_active", columnDefinition = "BOOLEAN DEFAULT TRUE")
    private boolean enquiryIsActive = true;

}

File: Computer-Seekho-.NET/Computer_Seekho_DN/Models/Enquiry.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace Computer_Seekho_DN.Models;

[Table("enquiry")]
[Index("StaffId", Name = "FKeud7g3l135dchk6imapeqd5ih")]
public partial class Enquiry
{
    [Key]
    [Column("enquiry_id")]
    public int EnquiryId { get; set; }

    [Column("closure_reason")]
    [StringLength(255)]
    public string? ClosureReason { get; set; }

    [Column("course_name")]
    [StringLength(255)]
    public string? CourseName { get; set; }

    [Column("enquirer_address")]
    [StringLength(255)]
    public string? EnquirerAddress { get; set; }

    [Column("enquirer_email_id")]
    [StringLength(255)]
    public string? EnquirerEmailId { get; set; }

    [Column("enquirer_mobile")]
    [StringLength(255)]
    public string? EnquirerMobile { get; set; }

    [Column("enquirer_name")]
    [StringLength(255)]
    public string? EnquirerName { get; set; }

    [Column("enquirer_query")]
    [StringLength(255)]
    public string? EnquirerQuery { get; set; }

    [Column("enquiry_counter")]
    public int? EnquiryCounter { get; set; }

    [Column("enquiry_date")]
    public DateOnly? EnquiryDate { get; set; }

    [Column("enquiry_is_active")]
    public bool? EnquiryIsActive { get; set; }

    [Column("follow_up_date")]
    public DateOnly? FollowUpDate { get; set; }

    [Column("student_name")]
    [StringLength(255)]
    public string? StudentName { get; set; }

    [Column("staff_id")]
    public int? StaffId { get; set; }

    [ForeignKey("StaffId")]
    [InverseProperty("Enquiries")]
    public virtual Staff? Staff { get; set; }
}
